docs/queries/queries.md

```diff
@@ -109,6 +109,25 @@ The same special operators are also automatically added on every column.
 * **gt** - Filter instances having values `Greater Than`.
 * **gte** - Filter instances having values `Greater Than Equal`.
 
+There is also a Q object which behaves like Q in django, without keywords. It is an alias of `and_()` of edgy.
+There is also a helper named: `or_()`. When empty `or_()` blocks in contrast to `and_()` and its alias `Q`.
+
+Note: this behaviour differs from sqlalchemy
```

Comment:
Missing dot at the end. 

Can we add the import of that Q here in the docs as well?

Reply:
yes